src/routes/Home/index.tsx

```diff
@@ -35,8 +56,11 @@ function Home(): ReactElement {
 
         <Card>
           <h2>Transactions to Sign</h2>
+          <PendingTxsList transactions={pendingTransactionsToDisplay} />
```

Comment:
It probably makes more sense to select the transactions in the component itself as it is much more modular.

Reply:
You mean passing the selected Safe to the widget and fetching from the Redux, right?
I agree with you and it will make the widget more modular indeed.

Reply:
You need not pass anything to the widget. You can select the transactions inside it.

Reply:
I decided to pass only `MAX_TXS_DISPLAY` to the widget

Reply:
I think it is a nice way to parametrize the widget with received props.

Reply:
Aren't we aiming to remaining modular though?